test/python/base/gf_dlr.py

```diff
@@ -103,6 +103,11 @@ def test_dlr_gfs_density(self):
         ref = np.exp(-e * beta) / (1 + np.exp(-e * beta))
         np.testing.assert_almost_equal(density(g_c), ref)
 
+        # block gf test
+        Bgw = BlockGf(name_list=['1', '2'], block_list=[g_c, g_c], make_copies=True)
+        for block, gf in Bgw:
+            np.testing.assert_almost_equal(density(g_c), ref)
```

Comment:
This is not really testing the any of the dlr BlockGf functions?

Also
```suggestion
        Bgw = BlockGf(name_list=['1', '2'], block_list=[g_c, g_c], make_copies=True)
        for block, gw in Bgw:
            np.testing.assert_almost_equal(density(gw), ref)
```